registration-processor/pre-processor/registration-processor-packet-validator-stage/src/test/java/io/mosip/registration/processor/stages/packet/validator/PacketValidateProcessorTest.java

```diff
@@ -245,41 +249,54 @@ public void setup() throws Exception {
-		Mockito.when(packetManagerService.getMetaInfo(anyString(), any(), any())).thenReturn(metamap);
-
+		Mockito.when(packetManagerService.getMetaInfo(any(), any(), any())).thenReturn(metamap);
 		Mockito.when(objectMapper.readValue(anyString(), any(Class.class))).thenReturn(new FieldValue("preRegistrationId", "12345"));
-
-
 	}
-	
+
 	@Test
-	public void PacketValidationSuccessTest() {
+	public void PacketValidationSuccessTest() throws PacketManagerException, ApisResourceAccessException, IOException, JsonProcessingException {
+		Map<String, String> metainfo1 = new HashMap<>();
+		metainfo1.put(JsonConstant.CREATIONDATE,"2023-10-17T03:01:09.893");
 		MessageDTO object = packetValidateProcessor.process(messageDTO, stageName);
-		assertTrue(object.getIsValid());
-		assertFalse(object.getInternalError());
+		ArgumentCaptor<InternalRegistrationStatusDto> argument = ArgumentCaptor
+				.forClass(InternalRegistrationStatusDto.class);
+		Mockito.verify(registrationStatusService,Mockito.atLeastOnce()).updateRegistrationStatus(argument.capture(), Mockito.any(),
+				Mockito.any());
+		Assert.assertEquals(LocalDateTime.parse(metainfo1.get(JsonConstant.CREATIONDATE)), argument.getAllValues().get(0).getPacketCreateDateTime());
+		Assert.assertTrue(object.getIsValid());
+		Assert.assertFalse(object.getInternalError());
 	}
-	
+
 	@Test
-	public void PacketValidationFailureTest() throws PacketValidatorException, ApisResourceAccessException, JsonProcessingException, RegistrationProcessorCheckedException, IOException, PacketManagerException {
-		registrationStatusDto.setRetryCount(1);
-		Mockito.when(packetValidator.validate(any(), any(),any())).thenReturn(false);
+	public void PacketValidationSuccessTestwithPacketCreatedDateTimeNull() throws PacketManagerException, ApisResourceAccessException, IOException, JsonProcessingException {
+		Map<String, String> metainfo = new HashMap<>();
+		metainfo.put(JsonConstant.CREATIONDATE,null);
+		Mockito.when(packetManagerService.getMetaInfo(any(), any(), any())).thenReturn(metainfo);
 		MessageDTO object = packetValidateProcessor.process(messageDTO, stageName);
-		assertFalse(object.getIsValid());
+		ArgumentCaptor<InternalRegistrationStatusDto> argument = ArgumentCaptor
+				.forClass(InternalRegistrationStatusDto.class);
+		Mockito.verify(registrationStatusService,Mockito.atLeastOnce()).updateRegistrationStatus(argument.capture(), Mockito.any(),
+				Mockito.any());
+		assertEquals(metainfo.get(JsonConstant.CREATIONDATE), argument.getAllValues().get(0).getPacketCreateDateTime());
+		assertTrue(object.getIsValid());
 		assertFalse(object.getInternalError());
 	}
-	
+
 	@Test
-	public void invalidSupervisorStatusTest() throws PacketValidatorException {
-		registrationStatusDto.setRetryCount(1);
-		regEntity=new SyncRegistrationEntity();
-		regEntity.setSupervisorStatus("REJECTED");
-		Mockito.when(syncRegistrationService.findByWorkflowInstanceId(any())).thenReturn(regEntity);
```

Comment:
why above 2 test cases are overwriten ? (invalidSupervisorStatusTest, PacketValidationFailureTest)

Reply:
added the test cases.